plugin-api/src/main/java/org/hyperledger/besu/plugin/services/metrics/LabelledSuppliedSummary.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright contributors to Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.plugin.services.metrics;
+
+import java.util.function.Supplier;
+
+/** The interface Labelled supplied summary. */
+public interface LabelledSuppliedSummary {
+  /**
+   * Labels.
+   *
+   * @param summarySupplier the summary supplier
+   * @param labelValues the label values
```

Comment:
I understand that, and I also found these comments just redundant, but since the build process requires us to put comments regardless the need of it, also in cases like this, where the code is self documenting, idk what it is better to do, wdyt?

Reply:
I'd suggest writing javadoc comments such that someone completely new to the code could read it and understand what the code is aiming to do

Reply:
Rewritten the javadoc to be more useful and removed it where the code is self-explanatory. 
Classes in this package are not public and using it requires at least some base knowledge of metrics and Prometheus, so the doc should take that for granted in this case.

Reply:
In general the javadoc requirement feels like an unnecessary barrier to me. While it is good to write something more meaningful rather than generated, I don't believe the forced comments are providing any actual value. Developers already add adhoc javadoc if it has value.